### PR TITLE
Add knative-eventing release-v0.22.0

### PR DESCRIPTION
**Pull request by lberk**

### CODE REVIEW

**Comment by lberk** [comment]
cc @matzew

File: ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v0.22.0-periodics.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    repo: knative-eventing
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: "47"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-knative-eventing-release-v0.22.0-47-e2e-aws-ocp-47-continuous
+  spec:
+    containers:
+    - args:
+      - --gcs-upload-secret=/secrets/gcs/service-account.json
```
Comment (matzew) [inline ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-release-v0.22.0-periodics.yaml:21]: @lberk should we add the slack reporter? like one other peridic jobs ?

File: core-services/image-mirroring/knative/mapping_knative_v0_22_quay [hunk 2]
```diff
@@ -0,0 +1,22 @@
+registry.ci.openshift.org/openshift/knative-v0.22.0:knative-eventing-test-event-flaker quay.io/openshift-knative/event-flaker:v0.22.0
+registry.ci.openshift.org/openshift/knative-v0.22.0:knative-eventing-test-event-flaker quay.io/openshift-knative/event-flaker:v0.22
```
Comment (matzew) [inline core-services/image-mirroring/knative/mapping_knative_v0_22_quay:2]: I checked, and this image is new, so I created a new repo here: https://quay.io/repository/openshift-knative/event-flaker?namespace=openshift-knative

and I added it to be updated (write) via our robot / bot account

Comment (matzew) [reply core-services/image-mirroring/knative/mapping_knative_v0_22_quay:2]: @lberk do you have access to our quay.io org?  If not, ping me offline

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**